dipy/reconst/shore.py

```diff
@@ -762,23 +761,24 @@ def shore_indices(radial_order, index):
     n_i = 0
     l_i = 0
     m_i = 0
-
+    """
     if n_c < (index + 1):
         msg = "The index %s is higher than the number of" % index
         msg += " coefficients of the truncated basis,"
         msg += " which is %s starting from 0." % int(n_c - 1)
         msg += " Select a lower index."
         raise ValueError(msg)
     else:
-        counter = 0
-        for l in range(0, radial_order + 1, 2):
-            for n in range(l, int((radial_order + l) / 2) + 1):
-                for m in range(-l, l + 1):
-                    if counter == index:
-                        n_i = n
-                        l_i = l
-                        m_i = m
-                    counter += 1
+    """
+    counter = 0
+    for n in range(radial_order + 1):
+        for l in range(0, n + 1, 2):
+            for m in range(-l, l + 1):
+                if counter == index:
+                    n_i = n
+                    l_i = l
+                    m_i = m
+                counter += 1
```

Comment:
I know nothing about this code but it looks like you could find `n_i`, `l_i` and `m_i` using /, mod and -. No?